source/includes/fundamentals/code-examples/poco.cs

```diff
@@ -0,0 +1,60 @@
+using MongoDB.Driver;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace TestRun.Fundamentals;
+
+public class Poco
+{
+    private static IMongoCollection<Clothing> _myColl;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+    
+    public static void Main(string[] args)
+    {
+        Setup();
+        
+        // start-insert
+        var doc = new Clothing() 
+        { 
+            Name = "Denim Jacket",
+            InStock = false,
+            Price = 32.99m,
+            ColorSelection = new List<string>() {"dark wash", "light wash"}
+        };
```

Comment:
Same for `Clothing`. Since you provide a property initializer list, you can omit the `()` after `new Clothing`. e.g. `new Clothing { Name = "Denim Jacket" }` rather than `new Clothing() { Name = "Denim Jacket" }`. The former is more idiomatic C#.